rpm/SOURCES/cntlm@.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=cNTLM authentication proxy - per user parameterized service
+After=network.target
+
+[Service]
+Type=simple
+User=%i
+ExecStart=/usr/libexec/cntlm-user
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
I guess this will still run it system-wide?

Reply:
At the system scope in the context of systemd however as the user set in the parameter; yes.

Reply:
Refactored to a proper user unit per earlier discussion.